app/controllers/people_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+  end
+
+
+    def show
+      @person = Person.find(params[:id])
+    end
```

Comment:
Looking good! Watch some of the indentation-- although Ruby doesn't mind if alignment is off, it can get confusing when you have lots of methods and you're trying to find a missing end or something :)